__tests__/app.test.js

```diff
@@ -174,3 +174,59 @@ describe('GET /api/articles/:article_id/comments', () => {
         })
     })
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('201: inserts a new comment into the specified article, and returns that newly added comment', () => {
+        const commentToAdd = {
+            body: 'this is a bunch of nonsense',
+            author: "butter_bridge",
+            votes: 0,
+            created_at: "2020-10-16T11:28:00.000Z"
+        }
+
+        const addedComment = {
+            body: 'this is a bunch of nonsense'
+        }
+        return request(app)
+        .post('/api/articles/4/comments') 
+        .send(commentToAdd) 
+        .expect(201)
+        .then(({body}) => {
+            const { comment } = body
+            expect(comment).toEqual(addedComment)
```

Comment:
If you consider what you would do with the return value in the front end? What information would you want to show?